Improve Spanish Translation (#658)
OWASP/owasp-masvs: Document-es/0x01-Foreword.md

Una cuestión importante que requiere el consenso de la industria es, ¿hasta dónde se debe llegar exactamente para proteger la confidencialidad e integridad de los datos? Por ejemplo, la mayoría de nosotros estaríamos de acuerdo en que una aplicación móvil debería verificar el certificado del servidor en una conexión TLS. Pero, ¿qué ocurre con el _certificate pinning_?, ¿el no implementarlo resultaría en una vulnerabilidad?, ¿debería ser este un requerimiento si una aplicación maneja datos sensibles?, ¿o es contraproducente?, ¿necesitamos cifrar los datos almacenados en bases de datos SQLite, a pesar de que el sistema operativo aísla la aplicación? Lo que es apropiado para una aplicación puede ser poco realista para otra. El MASVS es un intento de estandarizar estos requerimientos utilizando distintos niveles de verificación que se ajustan a los diferentes escenarios de amenaza.

> Además, la aparición del malware y las herramientas de administración remota han concienciado en que los propios sistemas operativos móviles tienen vulnerabilidades explotables, por lo que las estrategias de aislamiento se utilizan cada vez más para proporcionar protección adicional a los datos confidenciales y evitar la manipulación en el lado del cliente. Aquí es donde las cosas se complican. Las características de seguridad por hardware y las soluciones de aislamiento a nivel de sistema operativo, como _Android for Work_ y Samsung Knox, existen, pero no siempre están disponibles en diferentes dispositivos. A modo de _curita o tirita_, es posible implementar medidas de protección basadas en software - pero desafortunadamente, no hay estándares o procesos de prueba para verificar este tipo de protecciones.

Reviewer: ```suggestion
Además, la aparición del malware y las herramientas de administración remota han conseguido promover una mayor conciencia pública de que los propios sistemas operativos móviles tienen vulnerabilidades explotables, por lo que las estrategias de aislamiento se utilizan cada vez más para proporcionar protección adicional a los datos confidenciales y evitar la manipulación en el lado del cliente. Aquí es donde las cosas se complican. Las características de seguridad por hardware y las soluciones de aislamiento a nivel de sistema operativo, como _Android for Work_ y Samsung Knox, existen, pero no siempre están disponibles en diferentes dispositivos. Es posible implementar medidas de protección basadas en software a modo de parches - pero desafortunadamente, no hay estándares o procesos de prueba para verificar este tipo de protecciones.
```

> Como resultado, los informes de pruebas de seguridad de aplicaciones móviles están por todas partes: Por ejemplo, algunos reportan una falta de ofuscación o detección de _root_ en una aplicación Android como "fallo de seguridad". Por otra parte, las medidas como el cifrado de _strings_, la detección de depuradores o la ofuscación del flujo de control no se consideran obligatorias. Sin embargo, esta forma binaria de ver las cosas no tiene sentido porque la resistencia no es una proposición binaria sino que depende de las amenazas particulares en el dispositivo contra las que se quiere defender. Las protecciones de software no son inútiles, pero en última instancia pueden ser eludidas, por lo que nunca deben utilizarse como sustituto de los controles de seguridad.

Reviewer: ```suggestion
Como resultado, los informes de pruebas de seguridad de aplicaciones móviles están por todas partes: por ejemplo, algunos reportan una falta de ofuscación o detección de _root_ en una aplicación Android como "fallo de seguridad". Por otra parte, las medidas como el cifrado de _strings_, la detección de depuradores o la ofuscación del flujo de control no se consideran obligatorias. Sin embargo, esta forma binaria de ver las cosas no tiene sentido porque la resistencia no es una proposición binaria sino que depende de las amenazas particulares en el dispositivo contra las que se quiere defender. Las protecciones de software no son inútiles, pero en última instancia pueden ser eludidas, por lo que nunca deben utilizarse como sustituto de los controles de seguridad.
```

> El objetivo general del MASVS es ofrecer tanto una línea de base para la seguridad de las aplicaciones móviles (MASVS-L1) como también la inclusión de medidas de defensa en profundidad (MASVS-L2) y protecciones contra las amenazas del lado del cliente (MASVS-R). El MASVS está pensado para lograr lo siguiente:

Reviewer: ```suggestion
El objetivo general del MASVS es ofrecer tanto una línea base de referencia para la seguridad de las aplicaciones móviles (MASVS-L1) como también la inclusión de medidas de defensa en profundidad (MASVS-L2) y protecciones contra las amenazas del lado del cliente (MASVS-R). El MASVS está pensado para lograr lo siguiente:
```
